src/utils/parseFromHtml.js

```diff
@@ -97,14 +99,18 @@ export default async (inputHtml, inputUrl = '') => {
 
   const mainContent = stripUnwantedTags(mainContentSelected ?? html, unwanted)
 
-  const content = extractWithReadability(mainContent, bestUrl)
+  const mainContentAbsoluteUrls = normalizeUrls(mainContent, bestUrl)
+
+  const transformedContent = transformHtml(mainContentAbsoluteUrls, transform)
+
+  const content = extractWithReadability(transformedContent, bestUrl)
```

Comment:
Parsed so many document object. Maybe we should hold only one document when possible? Or try to use [linkedom cache](https://github.com/WebReflection/linkedom#cached-vs-not-cached=)(Need test)?